resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterScheduler.scala

```diff
@@ -374,6 +389,16 @@ private[spark] class MesosClusterScheduler(
     s"${frameworkId}-${desc.submissionId}${retries}"
   }
 
+  private[mesos] def getDriverPriority(desc: MesosDriverDescription): Float = {
+    val queueName =
+      desc.conf.get("spark.mesos.dispatcher.queue", config.DISPATCHER_QUEUE.defaultValueString)
+    if (queueName != config.DISPATCHER_QUEUE.defaultValueString) {
```

Comment:
Since we use this twice, shall we simplify a little like the following?
```scala
val defaultQueueName = config.DISPATCHER_QUEUE.defaultValueString
val queueName = desc.conf.get("spark.mesos.dispatcher.queue", defaultQueueName)
if (queueName != defaultQueueName) {
```

Reply:
Refactored :heavy_check_mark: